app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,24 @@
+# app/controllers/likes_controller.rb
```

Comment:
- [ ] Kindly remove this controller as it would have no matching routes after fixing the routes above.